targets/ChibiOS/RP_PICO_W_RP2040/Target_BlockStorage_RP2040FlashDriver.c

```
@@ -0,0 +1,159 @@
+#include <string.h>
+
+// RP2040 XIP flash base address
+#define RP2040_XIP_BASE 0x10000000U
+
+// RP2040 flash sector size (4KB)
+#define RP2040_FLASH_SECTOR_SIZE 4096U
+
+// Reference to ChibiOS EFL driver instance
+extern EFlashDriver EFLD1;
+
+bool RP2040FlashDriver_InitializeDevice(void *context)
```

Cppcheck (2.20.0)

[style] The function 'RP2040FlashDriver_InitializeDevice' is never used.

(unusedFunction)